charts/lamassu/templates/api-gateway-configmap.yml

```diff
@@ -71,6 +71,18 @@ data:
                         - name: backend
                           domains: ["*"]
                           routes:
+                            - match:
```

Comment:
I think we should add a comment explaining this change i.e. "Allow accessing VA services (CRL +OCSP) over HTTP without TLS. This is stated in RFC 5280 and it makes sense. Information only contains public non-confidential/sensitive data and contains signatures to prevent tampering and repudation"